GoogleApiWrappers/src/Google.Apis.Storage.v1.ClientWrapper/StorageClient.DownloadObject.cs

```diff
@@ -0,0 +1,165 @@
+﻿// Copyright 2015 Google Inc. All Rights Reserved.
+// Licensed under the Apache License Version 2.0.
+
+using Google.Apis.Download;
+using Google.Common;
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Object = Google.Apis.Storage.v1.Data.Object;
+
+namespace Google.Apis.Storage.v1.ClientWrapper
+{
+    public partial class StorageClient
+    {
+        // Note: documentation will be provided when we've worked out if we really want 16 methods here.
+        // See issue #21.
+
+        public void DownloadObject(string bucket, string sourceName, Stream destination)
```

Comment:
Wouldn't it be more correct for the second parameter to be called "Object name" or "Object path"?